tests/test_crypto/test_registry/test_crypto_registry.py

```diff
@@ -156,21 +160,21 @@ def test_wrong_spaces(self):
                 "malformed id", "path.to.module:CryptoClass"
             )
 
-    @pytest.mark.parametrize("special_character", string.punctuation.replace("_", ""))
+    @pytest.mark.parametrize("special_character", forbidden_special_characters)
     def test_special_characters(self, special_character):
         """Special characters are not allowed (only underscore)."""
         with pytest.raises(AEAException, match=self.MESSAGE_REGEX):
             aea.crypto.registries.register_crypto(
                 "malformed_id" + special_character, "path.to.module:CryptoClass"
             )
 
-    @pytest.mark.parametrize("digit", string.digits)
-    def test_beginning_digit(self, digit):
-        """Digits in the beginning are not allowed."""
-        with pytest.raises(AEAException, match=self.MESSAGE_REGEX):
-            aea.crypto.registries.register_crypto(
-                digit + "malformed_id", "path.to.module:CryptoClass"
-            )
+    # @pytest.mark.parametrize("digit", string.digits)
```

Comment:
I left this commented because not sure the item id regex change will be definitive.